databuilder/databuilder/utils/publisher_utils.py

```diff
@@ -0,0 +1,64 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+from os import listdir
+from os.path import isfile, join
+from typing import (
+    Iterator, List, Set,
+)
+
+from pyhocon import ConfigTree
+
+from databuilder.publisher.publisher_config_constants import PublisherConfigs
+
+
+def chunkify_list(records: List[dict], chunk_size: int) -> Iterator[List[dict]]:
+    """
+    Generator to evenly split the input list into chunks
+    """
+    for index in range(0, len(records), chunk_size):
+        yield records[index:index + chunk_size]
+
+
+def create_props_param(record_dict: dict, additional_publisher_metadata_fields: dict) -> dict:
+    """
+    Create a dict of all the params for a given record
+    """
+    params = {}
+
+    for k, v in {**record_dict, **additional_publisher_metadata_fields}.items():
```

Comment:
will remove/clean up some of these helpers in the next PR when removing UNQUOTED